components/ui/header.tsx

```diff
@@ -55,13 +55,13 @@ const Nav = () => (
   <Box className="translate-y-6">
     <Flex gap={4}>
       <Link href={"/courses"}>
-        <Button colorScheme="hollywood">Courses</Button>
+        <Button colorScheme="hollywood" shadow="5px 5px 0 black">Courses</Button>
       </Link>
       <Link href={"/tools"}>
-        <Button colorScheme="hollywood">Tools</Button>
+        <Button colorScheme="hollywood" shadow="5px 5px 0 black">Tools</Button>
```

Comment:
You're mixing between tailwind and Chakra here. use
`<Button colorScheme="hollywood" shadow={"5px 5px 0 black"}>Tools</Button>`
Although, doesn't the Button component already have that exact shadow style?